iConverter/Services/HistoryDataStore/HistoryDataStore.swift

```diff
@@ -11,8 +11,9 @@ import NSObject_Rx
 
 protocol HistoryDataStoreProtocol {
```

Comment:
I think that this should only save and load transactions so no need for `updateHistory` and `history`, if the user needs to update list, he can just call the `loadTransactions` after successful conversion